students/carlos_novoa/lesson03/mailroom.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python3
+import sys
+
+"""
+Lesson3 - Mailroom, Part 1
+"""
+
+_donors = [
+    [
+        'Jim Halpert',
+        [
+            1.00
+        ]
+    ],
+    [
+        'Pam Beesly',
+        [
+            1000.00, 2000.00, 3000.00
+        ]
+    ],
+    [
+        'Dwight Schrute',
+        [
+            2.00, 3.00
+        ]
+    ],
+    [
+        'Michael Scott',
+        [
+            10.0, 20.00, 30.00
+        ]
+    ],
+    [
+        'Andy Bernard',
+        [
+            500.00
+        ]
+    ]
+]
+
+
+def check_for_quit(action):
+    if action.lower() == 'quit':
+        donor_app()
+        return
+    else:
+        return
+
+
+def format_donation(donation):
+    donation = float(donation)
+    donation = "{0:.2f}".format(donation)
+    return donation
+
+
+def get_donation():
+    donation = input('Donation amount: ')
+    check_for_quit(donation)
+    return donation
+
+
+def add_donor(donor, donation):
+    _donors.append([donor, [float(donation)]])
+    return True
+
+
+def append_donor(donor, donation):
+    for d in _donors:
+        if donor.lower() == d[0].lower():
+            d[1].append(donation)
+            return True
+
+
+def thank_donor(donor_name, donation):
+    email_str = 'Dear {a},\nThank you for your generous ' \
+        + 'donation of ${b}.\nRegards\nUW Student'
+    print(email_str.format(a=donor_name, b=donation))
+    return
+
+
+def sort_donors(donors):
+    sorted_donors = list(donors)
+    sorted_donors.sort(key=lambda x: sum(x[1]))
+    sorted_donors.reverse()
+    return sorted_donors
+
+
+def get_donor_summary(donors):
+    summary = []
+    for i, donor in enumerate(donors):
```

Comment:
This can be `for donor in donors:` since you aren't using the index in your loop